model/model.py

```diff
@@ -93,8 +93,7 @@ def step(self, user_input: Optional[Direction]) -> None:
                 self._goto_next_state()
 
         elif self._curr_state == State.LEVEL_COMPLETE:
-            # TODO exit level somehow
-            pass #you can checkout -feature/end_level BUT YOU CAN'T NEVER LEAVE (guitar solo)
+            exit(-1)
```

Comment:
This quits the app before you can view your score & tips

Reply:
This is all hacky as shit, I don't want to add a User Input enum/class at this point or anything, I just want to be able to quit the game. I've updated this with another hack and removed an unnecessary `if` statement from the controller